cognite/neat/core/_data_model/models/entities/_single_value.py

```diff
@@ -549,3 +554,11 @@ def as_node_entity(self) -> DMSNodeEntity:
 
     def as_concept_entity(self) -> ConceptEntity:
         return ConceptEntity(prefix=self.prefix, suffix=self.suffix, version=self.version)
+
+
+class ContainerIndexEntity(ConceptualEntity):
+    type_: ClassVar[EntityTypes] = EntityTypes.container_index
+    prefix: _UndefinedType = Undefined
+    suffix: str
+    cursorable: bool = False
+    by_space: bool = Field(False, alias="bySpace")
```

Comment:
`bySpace` is currently not supported in the PySDK, which is why we cannot support it yet, but added it here as a placeholder ready once it is in the PySDK.